src/js/index.js

```diff
@@ -11,45 +11,64 @@
  * limitations under the License.
  */
 
-const wasm = require("./wasm");
+const nodejs =
+  typeof process !== "undefined" && process.versions && process.versions.node;
```

Comment:
Can make this slightly cleaner, don't need the `typeof` check
`const nodejs = process && process.versions && process.versions.node`